wandb/__init__.pyi

```diff
@@ -564,17 +559,15 @@ def log(
     For optimal performance, limit your logging to once every N iterations,
     or collect data over multiple iterations and log it in a single step.
 
-    ### The W&B step
-
-    With basic usage, each call to `log` creates a new "step".
+    Each call to `log` creates a new "step".
```

Comment:
```suggestion
    By default, each call to `log` creates a new "step".
```
something like this? there is an example below on how not to create a step with each call.